array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,11 @@
+/* exported take */
+function take(array, count) {
+  var takeArr = [];
+  if (count > array.length) {
+    return [];
```

Comment:
If the count is larger than the length, you don't want to return an empty array, you want to return the original array in full.